pytest_xmlcomp/plugin.py

```diff
@@ -74,16 +74,11 @@ def collect(self):
         Modify the XML files via a custom hook and parse the JSON.
         Returns: It returns XPath items which are found in the JSON file.
         """
-        from lxml import etree
         import json
-        originalxmlfile = self.fspath
+        xmlfile = self.fspath
+        print(self.fspath)
         pm = self.config.pluginmanager
-        modifiedxmlfile = pm.hook.pytest_transform_xml(xmlfile=originalxmlfile)
-        try:
-            tree = etree.parse(str(modifiedxmlfile[0]))
-        except etree.XMLSyntaxError:
-            print("XML Syntax Error in file %s" % self.fspath)
-            return False
+        tree = pm.hook.pytest_xmlcomp_transform_xml(xmlfile=str(xmlfile))
```

Comment:
Should we keep the exceptions handling? Maybe we should test a broken XML and JSON file just to handle this cases too.